lib/presentation/widgets/atoms/main_counter_button_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:animated_ripple/animated_ripple.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:water_tracker/custom_theme.dart';
+import 'package:water_tracker/generated/assets/assets.gen.dart';
+
+class MainCounterButtonWidget extends StatefulWidget {
```

Comment:
could it be stateless?

Reply:
Yes, if it is stateless i won't be able to create constants